packages/babel-generator/src/buffer.js

```diff
@@ -14,6 +14,7 @@ export default class Buffer {
     this._indent = format.indent.base;
     this.format = format;
     this.buf = "";
+    this.last = "";
```

Comment:
what's the value for maintaining this instance variable? Can't we just simply do `this.buf[this.buf.length -1]`?